modules/@apostrophecms/schema/index.js

```diff
@@ -130,7 +130,8 @@ module.exports = {
             return self.sortedDistinct(field.name, query);
           }
         });
-      }
+      },
+      def: ''
```

Comment:
With this change the required mechanism still triggers when it should (empty string is still empty as we tab off a field or attempt to save), and we don't get a false positive due to a transition from null to empty string.